survey/asker.go

```diff
@@ -173,13 +175,29 @@ func Ask() error {
 			},
 			Validate: survey.Required,
 		},
+		{
+			Name: "Clipboard",
+			Prompt: &survey.Confirm{
+				Message: "Copy reminder to clipboard?",
+				Default: true,
+			},
+		},
 	}
 
 	err := survey.Ask(qs, &answer)
 	if err != nil {
 		return err
 	}
 
-	fmt.Println(answer)
+	if answer.Clipboard {
+		if err := clipboard.WriteAll(answer.String()); err != nil {
+			log.Fatal(err)
```

Comment:
There have to return `err`

Reply:
Oh wow, sorry, I missed that :stuck_out_tongue: will fix right away